lib/tlscontext.h

```diff
@@ -44,6 +44,16 @@ typedef enum
   TVM_REQUIRED=0x0020,
 } TLSVerifyMode;
 
+typedef enum
+{
+  TSO_NONE,
+  TSO_NOSSLv2=0x0001,
+  TSO_NOSSLv3=0x0002,
+  TSO_NOTLSv1=0x0004,
+  TSO_NOTLSv11=0x0008,
+  TSO_NOTLSv12=0x0010,
+} TLSSslOptions;
+
```

Comment:
I tried to be consequent with the solution what was already there for the verify options. I did pretty much the same.
If my solution is bad, then verify options's is bad too.
If you'd like, I'd gladly correct both of them, but I think that should be a scope of a different PR, and this PR should focus only to implement the ssl-options() setting.
Opinions?